automatic/dbatools-library.powershell/update.ps1

```diff
@@ -0,0 +1,58 @@
+#import-module au
+
+. $PSScriptRoot\..\..\scripts\all.ps1
+
+$moduleName  = 'dbatools.library'
+
+function global:au_SearchReplace {
+    @{
+        "tools\chocolateyInstall.ps1" = @{
+            # We use ChocolateyPackageVersion to minimise changes to the package code -
+            # If using a patchfix version, use the moduleversion instead of the packageversion.
+            '(\s*\$moduleVersion\s*=\s*)(\$env:ChocolateyPackageVersion|(["'']?).+["'']?)' = "`${1}$(
+                if ($Latest.ModuleVersion -ne $Latest.Version) {
+                    $Quote = if ($Matches.'3') {'${3}'} else {"'"}
+                    $Quote + $Latest.ModuleVersion + $Quote
+                } else {
+                    '$env:ChocolateyPackageVersion'
+                }
+            )"
```

Comment:
After the discussion this morning with normalized and non-normalized versions, we should use the explicit version rather than the environment variables where we can. Happy to chat about this offline, as this has not been talked about more widely, yet.

Reply:
That sounds like a fun conversation! Is there a summary, somewhere?

Reply:
Tested the heck out of this, so hopefully we're good with the above.